core.go

```diff
@@ -75,6 +75,7 @@ func NewWithOpts(name, dir string, example Value, opts Opts, relationships ...st
 	c.opts = &opts
 	c.entryType = getCoreType(example)
 	c.logsDir = path.Join(dir, "logs")
+	c.indexFmt = fmt.Sprintf("%s0%dd", "%", 8)
```

Comment:
Correct me if I'm wrong, but isn't this just `c.indexFmt = "%08d"`? why the fmt.Sprintf?